# HG changeset patch
# User Betterbird <betterbird@betterbird.eu>
# Date 1704304833 -3600
# Parent  f5e565f9c9b2b3e3b1d1d2c8293d04dcbfbc3f96
Bug NNN4 - JS error in mailWindowOverlay.js:174

Fix
Uncaught TypeError: chromeBrowser.contentDocument.activeElement is null
InitEditMessagesMenu chrome://messenger/content/mailWindowOverlay.js:174
onpopupshowing chrome://messenger/content/messenger.xhtml:1

diff --git a/mail/base/content/mailWindowOverlay.js b/mail/base/content/mailWindowOverlay.js
--- a/mail/base/content/mailWindowOverlay.js
+++ b/mail/base/content/mailWindowOverlay.js
@@ -166,17 +166,17 @@ function file_init() {
 function InitEditMessagesMenu() {
   document.commandDispatcher.updateCommands("create-menu-edit");
 
   let chromeBrowser, folderTreeActive, folder, folderIsNewsgroup;
   let tab = document.getElementById("tabmail")?.currentTabInfo;
   if (tab?.mode.name == "mail3PaneTab") {
     chromeBrowser = tab.chromeBrowser;
     folderTreeActive =
-      chromeBrowser.contentDocument.activeElement.id == "folderTree";
+      chromeBrowser.contentDocument.activeElement?.id == "folderTree";
     folder = chromeBrowser.contentWindow.gFolder;
     folderIsNewsgroup = folder?.server.type == "nntp";
   } else if (tab?.mode.name == "mailMessageTab") {
     chromeBrowser = tab.chromeBrowser;
   } else {
     chromeBrowser = document.getElementById("messageBrowser");
   }
 
